### PR TITLE
Add known issue to NT

### PR DESCRIPTION
**Pull request by hnejra**

### CODE REVIEW

**Review by vadymv-mendix — APPROVED** [review]

File: content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-7-rn.md [hunk 1]
```diff
@@ -5,31 +5,44 @@ weight: 9
 description: "Native Template 7 release notes."
 ---
 
-## 7.0.13 {#714}
+## 7.0.14 {#714}
+
+**Release date: June 28, 2024**
+
+### Fixes
+We have fixed an issue with cookies encryption. In case the app still does not load correctly after updating to this version into a problem, please clear the app cache and reload the app.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-7-rn.md]:
```suggestion
We have fixed an issue with cookies encryption. If the app still does not load correctly after updating to this version into a problem, please clear the app cache and reload the app.
```

File: content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-7-rn.md [hunk 2]
```diff
@@ -5,31 +5,44 @@ weight: 9
...
+## 7.0.13 {#713}
 
 **Release date: June 5, 2024**
 
 ### Improvements
 
 * We enhanced encrypted file sync on Android.
 
-## 7.0.12 {#711}
+Known issue: In this version we addressed a cookie encryption security issue which turned out to be in compatible with the existing mechanism.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-7-rn.md]:
```suggestion
Known issue: In this version we addressed a cookie encryption security issue which turned out to be incompatible with the existing mechanism.
```

File: content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-8-rn.md [hunk 3]
```diff
@@ -21,6 +21,12 @@ description: "Native Template 8 release notes."
 
 * We upgraded the `@mendix/native` dependency to the latest compatible version.
 
+Known issue: In this version we addressed a cookie encryption security issue which turned out to be in compatible with the existing mechanism.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-8-rn.md]:
```suggestion
Known issue: In this version we addressed a cookie encryption security issue which turned out to be incompatible with the existing mechanism.
```

File: content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-8-rn.md [hunk 4]
```diff
@@ -92,6 +98,12 @@ description: "Native Template 8 release notes."
 * We fixed a synchronization issue that affected encrypting-decrypting files.
 * We upgraded the `@mendix/native` dependency to the latest compatible version.
 
+Known issue: In this version we addressed a cookie encryption security issue which turned out to be in compatible with the existing mechanism.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-8-rn.md]:
```suggestion
Known issue: In this version we addressed a cookie encryption security issue which turned out to be incompatible with the existing mechanism.
```

**Review by MarkvanMents — COMMENTED** [review]
Spotted these as I was looking at the issue you mentioned.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**